test/e2e/upgrade_test.go

```diff
@@ -206,3 +168,31 @@ func (tc *testContext) scaleWMCODeployment(desiredReplicas int32) error {
 
 	return err
 }
+
+// deployWindowsWorkloadAndTester tests if the Windows Webserver deployment is available.
+// This is achieved by creating a Job object that continuously curls the webserver every 5 seconds.
+// returns a tearDown func that must be executed to cleanup resources
+func (tc *testContext) deployWindowsWorkloadAndTester() (func(), error) {
+	// create a Windows Webserver deployment
+	deployment, err := tc.deployWindowsWebServer("win-webserver", nil)
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating Windows Webserver deployment for upgrade test")
+	}
+	// create a clusterIP service which can be used to reach the Windows webserver
+	intermediarySVC, err := tc.createService(deployment.Name, v1.ServiceTypeClusterIP, *deployment.Spec.Selector)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error creating service for deployment %s", deployment.Name)
+	}
+	// create a Job object that continuously curls the webserver every 5 seconds.
+	testerJob, err := tc.createLinuxJob(windowsWorkloadTesterJob, generateLinuxWorkloadTesterCommand(intermediarySVC.Spec.ClusterIP))
+	if err != nil {
+		return nil, errors.Wrapf(err, "error creating linux job %s", windowsWorkloadTesterJob)
+	}
+	// return a cleanup func
+	return func() {
+		// ignore errors while deleting the objects
+		_ = tc.deleteDeployment(deployment.Name)
+		_ = tc.deleteService(intermediarySVC.Name)
+		_ = tc.deleteJob(testerJob.Name)
```

Comment:
Do you really need `_ =`? It should be sufficient to leave that out given you have added the comment.

Reply:
It can be removed, yes; it's a matter of style. `_ =`  is the `go` way of saying  "explicitly ignore the returned error", and the inline comment complements that.